services/docker-compose.yml

```diff
@@ -343,6 +343,15 @@ services:
       - computational_services_subnet
     secrets: *dask_tls_secrets
 
+  efs-guardian:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/efs-guardian:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Task.Slot}}"
+    networks:
+      - default
+    environment:
```

Comment:
If there are any other env vars, they should all rest here

Reply:
Ye, no env vars yet, it is just a skeleton.